documentation/how-to/digitize.en.md

```diff
@@ -128,6 +128,14 @@ The reshape tool allows you to change line and polygon geometries by drawing sha
 result in a reshaped line following the edge of the drawn shape or a polygon with the drawn shaped
 used to clip or expand the geometry.
 
+## Reshape eraser tool
```

Comment:
This needs to be a 3rd level header (like the other tools); you'll also need to add the tool in the list of available tools at the beginning of the section.

Reply:
Great, thank you @nirvn!